src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -0,0 +1,137 @@
+# NTFS to Netex France conversion
+
+Version of the implementation: `1.0`
+
+## Introduction
+
+This document describes how a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) is transformed into a Netex profil France feed in Navitia Transit Model.
+
+The resulting ZIP archive is composed of:
+- a `arrets.xml` file containing the description of all stops (Quays and StopPlaces)
+- a `correspondances.xml` file containing all transfers between stops
+- a folder for each network containing
+  + TBD
+
+## Input parameters
+
+A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
+- `ParticipantRef` : The ParticipantRef code must be provided,
+- `StopProviderCode` : The code used to identify the provider in the stop_id generation.
+
+## File headers
+
+Each Netex file starts with a header containing some information generated by the program: 
+- `/PublicationDelivery/@version`: **x.y:FR-NETEX_nnnn-a.b-c** with
+  + `x.y`:  the version of Netex XSD of Netex (actually 1.09)
+  + `nnnn`:identification of the profile ("FRANCE", "COMMUN", "ARRET",
+"LIGNE", "RESEAU", "HORAIRE", "CALENDRIER" or "TARIF") 
+  + `a.b` is the profile version (2.1)
+  + `c` is the number of the local implementation (containing only integers and `.`). See at the top of the document.
+- `/PublicationDelivery/PublicationTimestamp`: generation date of the Netex Feed using ISO8601,
+- `/PublicationDelivery/ParticipantRef`: The value of the corresponding parameter to the conversion action.
+
+## arrets.xml
+
+A `stop_area` is considered monomodal if all the trips having stop_times referencing any of its stop_points have a physical_mode of the same "mode" Netex.
+
+### Quay
+
+Each `Quay` node corresponds to a NTFS `stop_point`.
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+Quay/@id | stops.txt | stop_id | see (1) below.
+Quay/@version | | | fixed value `any`.
+Quay/PublicCode | stops.txt | stop_code | This node may not be prensent if the stop_point has no `stop_code`.
+Quay/Name | stops.txt | stop_name | 
+Quay/Centroid/Location | stops.txt | stop_lat and stop_lon | see (2) below
+SiteRef | stops.txt | parent_station | If the parent_station is multimodal, see (3) below.
+TransportMode | | see (3) below
+tariffZones | stops.txt | fare_zone_id | The fare zone is prefixed by the `ParticipantRef` prefix with a `:` separator
+AccessibilityAssessment | stops.txt | equipment_id | This node is present only if the `equipment_id` is specified. see [AccessibilityAssessment]() below.
```

Comment:
this table is not well formatted on Github

Reply:
fixed